playbooks/maas-hummingbird.yml

```diff
@@ -0,0 +1,349 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Gather facts
+  hosts: hummingbird
+  gather_facts: "{{ gather_facts | default(true) }}"
+  pre_tasks:
+    - include: "common-tasks/maas_excluded_regex.yml"
+    #    - include: "common-tasks/maas_get_openrc.yml"
+    - name: Set the current group
+      set_fact:
+        maas_current_group: hummingbird
+
+  tasks:
+    - name: Copy over pip constraints
+      copy:
+        src: "files/pip-constraints.txt"
+        dest: "/tmp/pip-constraints.txt"
+      delegate_to: "{{ physical_host | default(ansible_host) }}"
+
+    - name: Retrieve openrc file
+      fetch:
+        src: /root/openrc
+        dest: /root/openrc
+        flat: true
+      delegate_to: "{{ item }}"
+      with_items: "{{ groups['hosts'] }}"
+      when:
+        - "'hosts' in groups"
+
+  post_tasks:
+    - name: Install swift pip packages to venv
+      pip:
+        name: "{{ maas_openstack_swift_pip_packages | join(' ') }}"
+        state: "{{ maas_pip_package_state }}"
+        extra_args: >-
+          --isolated
+          --constraint /tmp/pip-constraints.txt
+          {{ pip_install_options | default('') }}
+        virtualenv: "{{ maas_venv }}"
+      register: install_pip_packages
+      until: install_pip_packages|success
+      retries: 5
+      delay: 2
+      delegate_to: "{{ physical_host | default(ansible_host) }}"
+
+  vars_files:
+    - vars/main.yml
+    - vars/maas-hummingbird.yml
+  tags:
+    - maas-hummingbird
+
+- name: Install checks for hummingbird account
+  hosts: hummingbird
+  gather_facts: false
+  tasks:
+    - name: Install hummingbird account server checks
+      template:
+        src: "templates/rax-maas/hummingbird_account_server_check.yaml.j2"
+        dest: "/etc/rackspace-monitoring-agent.conf.d/hummingbird_account_server_check--{{ inventory_hostname }}.yaml"
+        owner: "root"
+        group: "root"
+        mode: "0644"
+      delegate_to: "{{ physical_host | default(ansible_host) }}"
+      notify:
+        - Restart rax-maas
+
+    - name: Install hummingbird account process checks
+      template:
+        src: "templates/rax-maas/hummingbird_account_process_check.yaml.j2"
+        dest: "/etc/rackspace-monitoring-agent.conf.d/hummingbird_account_process_check--{{ inventory_hostname }}.yaml"
+        owner: "root"
+        group: "root"
+        mode: "0644"
+      delegate_to: "{{ physical_host | default(ansible_host) }}"
+      notify:
+        - Restart rax-maas
+
+  handlers:
```

Comment:
This might causing issue on trusty related gates, would probably need to remove and test out ?

Refer it here: 
https://github.com/rcbops/rpc-maas/tree/stable/liberty/handlers
main.yml is an empty file. And this is causing gate failures:
https://rpc.jenkins.cit.rackspace.net/blue/organizations/jenkins/RPC-IRR_rpc-maas-master-trusty-mitaka/detail/RPC-IRR_rpc-maas-master-trusty-mitaka/561/pipeline